substrate/primitives/consensus/beefy/src/lib.rs

```diff
@@ -349,6 +349,19 @@ pub struct ForkVotingProof<Header: HeaderT, Id: RuntimeAppPublic, AncestryProof>
 	pub header: Header,
 }
 
+impl<Header: HeaderT, Id: RuntimeAppPublic> ForkVotingProof<Header, Id, OpaqueValue> {
+	/// Try to decode the `AncestryProof`.
+	pub fn try_into<AncestryProof: Decode>(
```

Comment:
@serban300  impl `TryInto` or `TryFrom` does not work here?

Reply:
I tried, but TryInto must return an `Error` and here it was better to return an `Option`.